modules/islandora_iiif/src/Plugin/views/style/IIIFManifest.php

```diff
@@ -188,6 +199,11 @@ public function render() {
           $label = $this->t("IIIF Manifest");
       }
 
+      /**
+       * @var \Drupal\taxonomy\TermInterface|null
+      */
+      $structured_text_term = $this->utils->getTermForUri($this->options['structured_text_term_uri']);
```

Comment:
Might make more sense to roll a getter (with memoization?) instead of passing it along. As in, the state already belongs to this instance (accessing from `$this`), no need to pass our state explicitly along to one of our methods that already has access to it.

Reply:
Could roll something like https://github.com/Islandora/islandora/compare/hocr...adam-vessey:islandora:hocr-fixes ?

Reply:
Thank you, I merged your fixes branch.